browser/css/style.css

```diff
@@ -5,95 +5,170 @@
 */
 
 .keybase-reply form {
-    position: absolute;
-    display: block !important;
-    margin-top: 2em;
-    background: #fff;
-    width: 500px;
-    white-space: normal !important;
-    font-size: 13px;
-    line-height: 1.5em;
-    border-radius: 4px;
-    background-color: #ffffff;
-    box-shadow: 3px 3px 13px 0 #a29e9e;
-    overflow: hidden;
+  overflow: hidden;
+  z-index: 9999;
+  position: absolute;
+  display: block !important;
+  background: #fff;
+  margin-top: 2em;
+  width: 500px;
+  border-radius: 4px;
+  background-color: #ffffff;
+  box-shadow: 3px 3px 13px 0 #a29e9e;
+  white-space: normal !important;
+  font-family: 'Open Sans', sans-serif !important;
+  font-size: 13px;
+  line-height: 1.5em;
+
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.keybase-reply label {
+  width: 100%;
+  display: inline-block;
+}
+
+.keybase-reply a {
+  font-size: inherit !important;
+}
+.keybase-reply a:hover {
+  text-decoration: none !important;
+}
+
+.keybase-body {
+  margin: 24px;
+  color: rgba(0, 0, 0, 0.75);
 }
 
 .keybase-error {
-    color: #ff4d61;
+  text-align: center;
+}
+
+.keybase-error .keybase-button {
+   margin: 8px 0 54px 0;
```

Comment:
Ugh my new editor settings keep switching back to 3-indent for CSS for some reason I can't figure out. Thanks for catching it.